tilequeue/query.py

```diff
@@ -82,6 +83,14 @@ def execute_query(conn, query, layer_datum, padded_bounds):
         cursor = conn.cursor(cursor_factory=RealDictCursor)
         cursor.execute(query)
         rows = list(cursor.fetchall())
+
+        # if we are debugging this layer, then print out the size of the
+        # results returned from the server.
+        if layer_datum.get('debug_query_size'):
+            logger = logging.getLogger('query')
+            logger.debug('Layer[%r]: %d bytes' %
+                         (layer_datum.get('name'), sys.getsizeof(rows)))
+
```

Comment:
The one gotcha is that I think the way the geometry gets encoded is into a reader type of object. The main thread that aggregates the results then explicitly [calls bytes()](https://github.com/tilezen/tilequeue/blob/0aed62cea12038230768f914569c1c7c9c677594/tilequeue/query.py#L175-L186) on it to convert it to bytes so that it can be pickled when passing it to the processors.

In any event, it's possible that this will report the correct number of bytes, but I'm not sure.